.github/workflows/github-actions-essential-ci.yml

```diff
@@ -344,3 +344,45 @@ jobs:
       - name: clean up
         run: ./build/github/cleanup-engflow-keys.sh
         if: always()
+  cockroach-microbench-ci:
```

Comment:
Unless this is going to be a mandatory job, I'd say this should be in its own new workflow instead of GitHub Actions Essential CI. Are you going to make this required to pass CI?

Reply:
This will be required in future (~1 month after) but not in this PR.